app/src/main/res/values/strings.xml

```diff
@@ -436,4 +436,5 @@
   <string name="coming_soon">Coming Soon</string>
   <string name="recommended_stories">Recommended Stories</string>
   <string name="stories_for_you">Stories For You</string>
+  <string name="question_player_activity_title">Question Player</string>
```

Comment:
I missed this earlier. The string is incorrect, it should be `Practice Mode`

Reply:
@rt4914 Done. Just one check failing.

Reply:
@rt4914 Passed all tests. PTAL.